exasol_advanced_analytics_framework/resources/outputs/create_query_loop.sql

```diff
@@ -1,4 +1,4 @@
-CREATE OR REPLACE LUA SCRIPT "AAF_QUERY_LOOP"(json_str) RETURNS TABLE AS
+CREATE OR REPLACE LUA SCRIPT "AAF_RUN_QUERY_HANDLER"(json_str) RETURNS TABLE AS
```

Comment:
We check this in, because, we package it into the python package, such that the user of the installer CLI can use it without having LUA and amalg installed. To be able to package it into the package as well for package installs directly from the repo, we need it currently in the repo. Later, when this project has more stabilized we probably, we can go over and only generate in the CI and add it to the package during release artifact build.

Reply:
:heavy_check_mark: